MIGRATION.md

```diff
@@ -0,0 +1,106 @@
+<!-- shared header  START -->
+
+<p align="center">
+  <a href="https://www.contentful.com/developers/docs/references/content-management-api/">
+    <img alt="Contentful Logo" title="Contentful" src="images/contentful-icon.png" width="150">
+  </a>
+</p>
+
+<h1 align='center'>Content Management API</h1>
+
+<h3 align="center">Migration</h3>
+
+<p align="center">
+  <a href="README.md">Readme</a> · 
+  <a href="MIGRATION.md">Migration</a> · · 
+  <a href="CONTRIBUTING.md">Contributing</a>
+</p>
+
+<p align="center">
+  <a href="https://www.contentful.com/slack/">
+    <img src="https://img.shields.io/badge/-Join%20Community%20Slack-2AB27B.svg?logo=slack&maxAge=31557600" alt="Join Contentful Community Slack">
+  </a>
+</p>
+
+<!-- shared header  END -->
+
+# Migration information
+
+- [Migration information](#migration-information)
+  - [Migration to version 12.x](#migration-to-version-12x)
+    - [Breaking changes](#breaking-changes)
+      - [Node.js core modules](#nodejs-core-modules)
+      - [Pre-bundled code](#pre-bundled-code)
+        - [Webpack 5](#webpack-5)
+        - [Rollup](#rollup)
+    - [Improvements](#improvements)
+      - [Tree shaking](#tree-shaking)
+      - [Module support and package configuration](#module-support-and-package-configuration)
+      - [Testing framework](#testing-framework)
+  - [Security](#security)
+    - [Removal of eval](#removal-of-eval)
+
+From version 3.0.0 onwards, you can access documentation for a specific version by visiting `https://contentful.github.io/contentful.js/contentful/<VERSION>`.
+
+You can upgrade to a major version using `npm update contentful`
```

Comment:
This copy-pasted from contentful.js. Should be updated as it links to the wrong repo/package

Reply:
great catch!